app/globals.css

```
@@ -1173,6 +1173,9 @@ body.light * {
   animation: img-breathe 3s ease-in-out infinite;
 }
 
+/* Import CodeBlock styles */
+@import '../components/chat/CodeBlockStyle.module.css';
```

Biome (2.1.2)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)